.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+## Description
+[What is wrong? What the expected and current behaviours are?]
+
+
+## Code to reproduce
```

Comment:
## Reproduction
[If possible, please include example code and image(s) to reproduce this issue.]